### PR TITLE
travis: run integration tests via script

### PR DESCRIPTION
**Pull request by hunchback**
eliminates the in .travis.yam duplication of the integration test script.

if the change passes CI phase of integration test then the change is ok

### CODE REVIEW

File: run-integration-tests-local.sh [hunk 1]
```diff
@@ -9,11 +9,14 @@
...
-export DOCKER_NAMESPACE=m4d-system
+set -x
+set -e
+
+export DOCKER_HOSTNAME=${DOCKER_HOSTNAME:=kind-registry:5000}
+export DOCKER_NAMESPACE=${DOCKER_NAMESPACE:=m4d-system}
 export HELM_EXPERIMENTAL_OCI=1
 
-make kind-setup
+make kind
```
Comment (froesef) [inline run-integration-tests-local.sh:19]: Why do you upgrade this to kind? Do we need istio somewhere? This will increase the runtime quite a bit..

Comment (roee88) [reply run-integration-tests-local.sh:19]: opened a related issue https://github.com/IBM/the-mesh-for-data/issues/132

File: .travis.yml [hunk 2]
```diff
@@ -86,17 +86,7 @@ jobs:
...
-    - make docker
-    - make -C test/services docker # Build test services
-    - make cluster-prepare-wait || travis_terminate 1;
-    - make -C secret-provider configure-vault
-    - make -C secret-provider deploy || travis_terminate 1;
-    - make -C manager deploy_it || travis_terminate 1;
-    - make -C manager wait_for_manager || travis_terminate 1;
-    - make helm || travis_terminate 1;
-    - make -C manager run-integration-tests || travis_terminate 1;
+    - ./run-integration-tests-local.sh
```
Comment (roee88) [inline .travis.yml:89]: why not in `/hack`?

Comment (hunchback) [reply .travis.yml:89]: agree will move it

File: run-integration-tests-local.sh (hunk 1 above)
Comment (hunchback) [reply run-integration-tests-local.sh:19]: ok, will revert change, and I have assigned myself to the issue

**Comment by roee88** [comment]
Looking at the failed CI, it tries to push to ghcr.io. @hunchback can you please fix this?

**Comment by hunchback** [comment]
@roee88 - have rebased

**Review by roee88 — APPROVED** [review]
Thanks 👍

**Pull request merged**